package.json

```diff
@@ -114,7 +114,12 @@
   "stability": "stable",
   "jsii": {
     "outdir": "dist",
-    "targets": {},
+    "targets": {
+      "python": {
```

Comment:
Why do we have a target for Python but not for Node here?

Reply:
That is a great question I do not know for sure but I assume its because JSII is configured by default to create a distribution for Node. I know that by default when I run `yarn build` it will create a JS distribution folder and only when I added the python target did it also create the python distribution.

Reply:
Yeah based on the JSII [docs](https://aws.github.io/jsii/user-guides/lib-author/configuration/#targets) it seems like the targets are just the the other languages you would like to create artifacts for. It seems  that by default a Node artifact is created and only if you specify it then other artifacts will be generated.